### ISSUE TITLE
esp-radar是否只支持含有ws2812-led-driver的开发板？

### ISSUE THREAD

**Issue by Yanc14** [espressif/esp-csi#141]

**Comment by MacChu0315-Espressif** [comment]
No, the LED is only used for a simple and intuitive display of the detection results. If your device doesn't have an LED, you can still check the detection results through alternative methods.

Is there any specific issue you're encountering during usage?
